pages/card/card.css

```diff
@@ -0,0 +1,239 @@
+    background-color: #fff;
+    color: rgb(0, 0, 0);
+    border-radius: 10px;
+    overflow: hidden;
+    box-shadow: rgba(130, 130, 130, 0.95) 0px 10px 20px, rgba(130, 130, 130, 0.950) 0px 6px 6px;
+    transition: 0.5s;
+}
+
+.card-content{
+    margin-bottom: 5rem; 
+}
+
+.vertical-card-container {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+    gap: 4rem;
+}
+
+.card:hover{
+    box-shadow: rgba(130, 130, 130, 0.95) 0px 15px 25px, rgba(130, 130, 130, 0.950) 0px 10px 10px;
+}
+
+.card-info {
+    padding: 0.5rem 1rem;
+    position: relative;
+    width: 100%;
+}
+
+.card-title {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-start;
+    position: relative;
+}
+
+.card-title>div>h3{
+    font-size: 1.5rem;
+}
+
+.card-title>div>p{
+    color:rgb(110, 110, 110);
+    font-size: 0.8rem;
+}
+
+.card-title>i {
+    padding-top: 0.5rem;
+    text-decoration: 1px solid black;
+}
+
+.price{
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    gap: 1rem;
+}
+
+.current-price{
+    font-size: 1.2rem;
+}
+
+.actual-price{
+    color: rgb(90, 90, 90);
+    text-decoration: line-through;
+}
+
+.discount-percentage{
+    color: #499942;
+}
+
+.bottom-btn.cart{
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    padding: 0.5rem 0.6rem;
+}
+
+.add-cart{
+    border: none;
+    color: #fff;
+    background-color: rgb(0, 0, 0);
+    width: 100%;
+    font-size: 1.2rem;
+    padding: 0.5rem 1rem;
+    border-radius: 5px;
+}
+
+.card-badge{
+    position: absolute;
+    display: inline-block;
+    margin-top: 0.8rem;
+    border-top-right-radius: 5px;
+    border-bottom-right-radius: 5px;
+    background-color: #E67A00;
+    color:rgb(255, 255, 255);
+    padding : 0.1rem 1rem;
+}
+
+.horizontal-card-container{
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.card-horizontal{
+    display: flex;
+    flex-direction: row;
+    border-bottom: 2px solid rgb(180, 180, 180);
+    width: 550px;
+}
+
+.quantity{
+    margin-top: 1rem;
+}
+
+.quantity>.minus,.add{
+    border: 1px solid rgb(180, 180, 180);
+    background: none;
+    padding: 0.3rem 0.5rem;
+    border-radius: 50%;
+}
+
+/* input */
+.quantity>.count{
+    padding: 0.5rem 0.5rem;
+    width: 40px;
+    text-align: center;
+}
+input::-webkit-outer-spin-button,
+input::-webkit-inner-spin-button {
+-webkit-appearance: none;
+margin: 0;
+}
+input:focus, textarea:focus, select:focus{
+    outline: none;
+}
+
+.horizontal-btn{
+    display: flex;
+    margin: 1rem 5rem;
+    align-items: center;
+    justify-content: space-around;
+}
+
+.remove-btn{
+    border: none;
+    background: none;
+    font-size: 1rem;
+    font-weight: bold;
+}
+
+.remove-btn:hover{
+    color:rgb(90, 90, 90)
+}
+
+.later-btn{
+    border: none;
+    background: none;
+    font-size: 1rem;
+    font-weight: bold;
+    color: blue;
+}
```

Comment:
you can try to avoid code repetition here by creating new class . like this
```css
.btn{
    border: none;
    background: none;
    font-size: 1rem;
    font-weight: bold;
}

.later-btn{
 color : blue; /*  you should declare variable for the color */
}
```